Dmf/Modules.Library/Dmf_UdeClient.c

```diff
@@ -1691,6 +1821,105 @@ Return Value:
 }
 #pragma code_seg()
 
+#pragma code_seg("PAGE")
+_IRQL_requires_max_(PASSIVE_LEVEL)
+VOID
+DMF_UdeClient_DeviceEndpointInformationGet(
+    _In_ DMFMODULE DmfModule,
+    _In_ UDECXUSBENDPOINT Endpoint,
+    _Out_opt_ UDECXUSBDEVICE* UdecxUsbDevice,
+    _Out_opt_ UCHAR* Address
+    )
+/*++
+
+Routine Description:
+
+    Get the address from a given endpoint.
+
+Arguments:
+
+    DmfModule - This Module's handle.
+    Endpoint - The given endpoint.

```

Comment:
an argument is missing here.

Reply:
i don't see anything missing...

````
_IRQL_requires_max_(PASSIVE_LEVEL)
VOID
DMF_UdeClient_DeviceEndpointInformationGet(
    _In_ DMFMODULE DmfModule,
    _In_ UDECXUSBENDPOINT Endpoint,
    _Out_opt_ UDECXUSBDEVICE* UdecxUsbDevice,
    _Out_opt_ UCHAR* Address
    )
````

Gets information about a given endpoint.

##### Returns

None

##### Parameters
Parameter | Description
----|----
DmfModule | An open DMF_UdeClient Module handle.
Endpoint | The given endpoint.
UdecxUsbDevice | The device to which the endpoint is attached.
Address | The address of the endpoint.

Reply:
fixed